src/Dev/SapphireTest.php

```diff
@@ -563,6 +570,10 @@ protected function tearDown(): void
 
         // Call state helpers
         static::$state->tearDown($this);
+
+        // Reset max_execution_time in case some code or a unit test changed it,
+        // either via ini_set() or set_time_limit()
+        ini_set('max_execution_time', $this->origMaxExecutionTime);
```

Comment:
Don't we still need to reset `set_time_limit()`?

Reply:
set_time_limit() is an shorthand alias for ini_set('max_execution_time').  It seems slightly less confusing just not using the alias.

I've tested locally that it still fixes the issue that was seen in CI

Reply:
I can't find anything in the docs of `set_time_limit()` that supports it being an alias, but I'll accept "I've tested locally" as good enough.